src/frontend/js/data/CourseProductProvider/index.spec.tsx

```diff
@@ -24,6 +24,7 @@ describe('useCourseProduct', () => {
   });
 
   it('throws an error if it is not used within a CourseProductProvider', () => {
+    jest.spyOn(console, 'error').mockImplementation(noop);
```

Comment:
Are you sure that you do not need to unmock this at the bottom of this test ? ( are there any risk that it hides error of other following test suites ? )

Reply:
This the purpose of the `jest.restoreAllMocks` within the `afterEach`

Reply:
Right!